pkg/ottl/contexts/internal/errors.go

```diff
@@ -0,0 +1,22 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package internal // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/internal"
+
+import "fmt"
+
+const (
```

Comment:
Can we make these local to the package they refer to? It feels like each package should own its own reference.

Reply:
I could for the contexts that don't share anything in`internal`, but `resource`, `metric`, `scope`, and `span` all have their business logic in `interrnal`.

Reply:
We could move the errors to the files for each, but it's internal anyway, so it's fine for now. We can revisit this if something changes later.